test/test_nn.py

```diff
@@ -8956,6 +8956,37 @@ def __call__(self, input):
     input_size=(4, 16),
     fullname='AdaptiveLogSoftmax'))
 
+class _LayerNormWithNonDefaultWeight(nn.LayerNorm):
+    def __init__(self, normalized_shape, eps, elementwise_affine):
+        super().__init__(normalized_shape, eps, elementwise_affine)
+        self.weight.data = 0.1 * torch.ones_like(self.weight.data)
+
+class TestLayerNormWithNonDefaultWeight(NewModuleTest):
+    def __init__(self):
+        super().__init__(
+            constructor=lambda: _LayerNormWithNonDefaultWeight((16,), 1e-5, True),
+            input_size=(4, 4, 16),
+            check_gradgrad=True,
+            fullname='LayerNormWithNonDefaultWeight')
+
+    def compare_two_gradients(self, test_case, module, input):
+        num_threads = torch.get_num_threads()
```

Comment:
Why is this needed? The accumulation operations are usually more precise when you have multiple threads actually.

Reply:
Removed